docs/reference_guide.md

```diff
@@ -3181,6 +3182,37 @@ dropped privs to tcpdump
 10:23:45.823229 IP 100.101.2.146.53 > 192.168.0.23.46158: 45799 1/0/0 A 100.100.45.106 (60)
 ```
 
+## 32. `pton()`: Convert text IP address to byte array
+
+Syntax: `pton([int af, ]char* addr)`
+
+This converts a text representation of an IPv4 or IPv6 address to byte array. 
+`pton` will parse `addr` as address type given by `af`. If `af` is not specified, `pton` assumes `addr` as IPv4 type.
+`pton` comes in handy when we need to select packets with certain IP addresses.
+
+Examples:
+
+```
+# bpftrace --include /usr/include/arpa/inet.h \
```

Comment:
Ideally we do not require import header files at all. BTF should give us nearly everything we need. In this case I think we can omit the `AF_INET[4|6]` optional params by inferring AF from the string literal.